citation.php

```diff
@@ -96,13 +96,37 @@ function get_citation_by_id($id)
 // build the full dcoid
 $dx = "http://dx.deepcarbon.net/".$dcoid;
 
+// helper funciton to format an author name into LAST FIMI format.
+function format_name($name)
+{
+    $comma_pos = strpos($name, ",");
+    if ( $comma_pos != FALSE)
+    {
+        $last_name = substr($name, 0, $comma_pos);
+        $fm_names = substr($name, $comma_pos); 
+    }
+    else 
+    {
+        $space_pos = strrpos($name, " ");
```

Comment:
Good catch. Should just ignore anything that doesn't start with alphanumeric I think. I changed Marshall's profile to remove the nickname since there's already a field for that.